docs/source/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+Getting Started
+============
+
+
+Glossary
+--------
+
+- A ``DataSource`` represents a way to retrieve data from database. Typically, this corresponds to a table in the database. Yet, it could also be a more elaborate object. See the section on 'Alternative ``DataSource`` s' for more detail.
+
+- A ``Constraint`` captures a concrete expectation between either two ``DataSource`` s or a single ``DataSource`` and a reference value.
+
+- A ``Requirement`` captures all ``Constraint`` s between two given ``DataSource`` s or all ``Constraint`` s within a single ``DataSource``. If a ``Requirement`` refers links to two ``DataSource`` s, it is a ``BetweenRequirement``. If a ``Requirement`` merely refers to a single ``DataSource``, it is a ``WithinRequirement``.
+
+- Conceptually, a 'specification' captures all ``Requirement`` s against a database. In practice that means it is usually a separate python file which:
+  - gathers all relevant ``Requirement`` s
+  - turns these ``Requirement`` s' ``Constraint`` s into individual tests
+  - can be used to be called with pytest
+
+
+Creating a specification
+------------------------
+
+In order to get going, you might want to use the following snippet in a new python file.
+This file will represent a specification.
+
+.. code-block:: python
+
+    import pytest
+    import sqlalchemy as sa
+    from dbcheck.pytest_integration import collect_data_tests
+
+
+    @pytest.fixture(scope="module")
+    def datajudge_engine():
+        # TODO: Adapt connection string to database at hand.
+        return sa.create_engine("your_connection_string")
+
+
+    # TODO: Insert Requirement objects to list.
+    requirements = []
+
+    test_constraint = collect_data_tests(requirements)
+
+This file will eventually lead as an input to pytest. More on that in the section 'Testing a specification'.
+
+
+Specifying Constraints
+----------------------
+
+In order to discover possible ``Constraint`` s, please investigate the ``_add_*_constraint`` methods
+for `BetweenRequirement <https://datajugde.readthedocs.io/en/latest/api/datajudge.requirements.html#datajudge.requirements.BetweenRequirement>`_
+and `WithinRequirement <https://datajugde.readthedocs.io/en/latest/api/datajudge.requirements.html#datajudge.requirements.WithinRequirement>`_
+respectively.
+
+These methods are meant to be mostly self-documenting through usage of expressive parameters.
+
+Note that most ``Constraint`` s will allow for at least one ``Condition``. A ``Condition``
```

Comment:
```suggestion
Note that most ``Constraint``s will allow for at least one ``Condition``. A ``Condition``
```

Was that space intended or not? This space between the ending `` and the s to indicate plural happens quite a bit here. Didn't want to mark all of them in case it was intended.